### PR TITLE
[WIP] Remove gpaddon from PXF PR pipeline

### PR DESCRIPTION
**Pull request by shivzone**

### CODE REVIEW

File: concourse/pipelines/pxf_pr_pipeline.yml [hunk 1]
```diff
@@ -39,18 +44,39 @@ resources:
...
+    regexp: centos6/sigar-rhel6_x86_64-(.*)\.targz
+
+- name: python-centos6
+  type: gcs
   source:
-    branch: {{gpaddon-git-branch}}
-    private_key: {{gpaddon-git-key}}
-    uri: {{gpaddon-git-remote}}
+    bucket: gp-internal-artifacts
+    json_key: ((gcs-key))
```
Comment (denalex) [inline concourse/pipelines/pxf_pr_pipeline.yml]: ideally we should not use any keys and use only public artifacts

Comment (shivzone) [reply concourse/pipelines/pxf_pr_pipeline.yml]: That was an outdated version

File: concourse/README.md [hunk 2]
```diff
@@ -31,8 +31,9 @@ pushd ~/workspace/gp-continuous-integration && git pull && popd
 fly -t ud set-pipeline \
     -c ~/workspace/pxf/concourse/pipelines/pxf_pr_pipeline.yml \
     -l ~/workspace/gp-continuous-integration/secrets/gpdb-release-secrets.dev.yml \
-    -l ~/workspace/gp-continuous-integration/secrets/gpdb_5X_STABLE-ci-secrets.yml \
-    -v folder-prefix=dev/pivotal-default -v test-env=dev -v gpdb-git-branch=5X_STABLE \
+    -l ~/workspace/gp-continuous-integration/secrets/gpdb_master-ci-secrets.dev.yml \
```
Comment (frankgh) [inline concourse/README.md]: Can we stop using secrets to set this pipeline?

Comment (shivzone) [reply concourse/README.md]: Yes, will update

File: concourse/README.md [hunk 3]
```diff
@@ -31,9 +32,7 @@ pushd ~/workspace/gp-continuous-integration && git pull && popd
 fly -t ud set-pipeline \
     -c ~/workspace/pxf/concourse/pipelines/pxf_pr_pipeline.yml \
     -l ~/workspace/gp-continuous-integration/secrets/gpdb-release-secrets.dev.yml \
```
Comment (denalex) [inline concourse/README.md]: is this secret file needed only for github access token ? how does gpdb pr pipeline handle it ?

Comment (shivzone) [reply concourse/README.md]: pxf-bot-access-token is required for the PR pipeline to function. bot access token is the standard approach for PR pipelines, we can look at moving this out to a PXF specific file though

**Comment by frankgh** [comment]
Just a quick note, we can set the jobs to `public: true` once we have eliminated all private keys/artifacts etc.

File: concourse/scripts/pxf_common.bash [hunk 4]
```diff
@@ -42,11 +42,29 @@ function run_regression_test() {
...
 }
 
+function build_install_gpdb() {
+    pushd gpdb_src
+        CC=$(which gcc) CXX=$(which g++) ./configure \
+        --with-perl \
+        --with-python \
+        --with-libxml \
+        --with-zstd \
+        --enable-mapreduce \
```
Comment (frankgh) [inline concourse/scripts/pxf_common.bash]: do we need mapreduce?

**Pull request merged**